tokio-tests/src/lib.rs

```diff
@@ -1,7 +1,7 @@
 #![cfg(not(windows))]
-
-#[cfg(feature = "tokio-support")]
+#[cfg(test)]
 mod tests {
```

Comment:
I think that now that the test is in a separate crate and run selectively, the whole `cfg` and module thing is no longer needed.

Reply:
I'm not sure how this is usually done but I think the `mod tests` is still useful because else all the imports will emit an unused import warning.

Reply:
You're right there. I was thinking in the line of files living in `tests`, which don't need this as they are always being tests.